### PR TITLE
Understanding Identity Management with Entando using Keycloak

### PR DESCRIPTION
**Pull request by walkerippon**

### CODE REVIEW

File: posts/understand-entando-identity-management-using-keycloak.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# Understanding Entando Identity Management
+
+The Keycloak service powers all authentication and authorization for Entando. Entando applications rely on a Keycloak instance that is either externally installed or specific to an application.
+
+## Using Keycloak for Entando Authentication
+
+Entando implements Keycloak as a central authentication point to provide a unified view of identity. Keycloak acts as an abstraction layer to the underlying Identity Provider (IDP), allowing Entando to integrate into other IDPs without modifying the source. This schema increases portability, which is beneficial for configuring all authentication for an Entando application in one place.
```
Comment (dsharpe) [inline posts/understand-entando-identity-management-using-keycloak.md:21]: The last sentence is a little confusing to me. I'm not sure what "This schema" is referencing. Or is it just a typo and should be "scheme"?

**Review by dsharpe — CHANGES_REQUESTED** [review]

File: posts/understand-entando-identity-management-using-keycloak.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+https://github.com/entando/entando-keycloak-plugin
+
+### Enable Keycloak on an Entando Instance
+
+https://github.com/entando/entando-keycloak-plugin/wiki
+
+### Entando Identity Management System
+
+https://developer.entando.com/v7.1/docs/consume/identity-management.html#logging-into-your-keycloak-instance
```
Comment (pkrish123) [inline posts/understand-entando-identity-management-using-keycloak.md:55]: After this, I would include a conclusion and call to action.

A call to action would be something along the lines of:

```
For more information on how Ippon Technologies can help your organization utilize Entando for all of your application development needs, contact [sales@ipponusa.com](mailto:sales@ipponusa.com).

**Review by pkrish123 — CHANGES_REQUESTED** [review]

File: posts/understand-entando-identity-management-using-keycloak.md (hunk 2 above)
Comment (pkrish123) [reply posts/understand-entando-identity-management-using-keycloak.md:55]: Sum up what you've included as the content during the blog!

**Review by pkrish123 — CHANGES_REQUESTED** [review]

File: posts/understand-entando-identity-management-using-keycloak.md (hunk 2 above)
Comment (walkerippon) [reply posts/understand-entando-identity-management-using-keycloak.md:55]: During the earlier stages of making this blog I had a brief summary but was advised against it for the purpose of being redundant. Let me know of any other suggestions though!

**Review by pkrish123 — APPROVED** [review]

**Review by dsharpe — APPROVED** [review]

**Pull request merged**